<ui version="4.0" stdsetdef="1" >
  <author>Sebastian Trueg</author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>base_K3bAudioRippingOptionWidget</class>
  <widget class="QWidget" name="Form1" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>436</width>
        <height>182</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <item row="0" column="0" >
        <widget class="Q3GroupBox" name="groupBox2" >
          <property name="title" >
            <string>Filetype</string>
          </property>
          <layout class="QHBoxLayout" >
            <item>
              <widget class="KComboBox" name="m_comboFileType" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>1</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QToolButton" name="m_buttonConfigurePlugin" >
                <property name="text" >
                  <string>...</string>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>Configure Plugin</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item rowspan="2" row="0" column="1" colspan="1" >
        <widget class="Q3GroupBox" name="groupBox3" >
          <property name="title" >
            <string>Options</string>
          </property>
          <layout class="QVBoxLayout" >
            <item>
              <widget class="QCheckBox" name="m_checkCreatePlaylist" >
                <property name="text" >
                  <string>Create m&amp;3u playlist</string>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>Create playlist for the ripped files</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;p>If this option is checked K3b will create a playlist of the ripped files
which can be used with programs like xmms or noatun.
&lt;p>You may use the special strings to give the playlist a unique filename.</string>
                </property>
              </widget>
            </item>
            <item>
              <layout class="QHBoxLayout" >
                <item>
                  <spacer name="spacer1_2" >
                    <property name="sizeHint" >
                      <size>
                        <width>20</width>
                        <height>16</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item>
                  <widget class="QCheckBox" name="m_checkPlaylistRelative" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="text" >
                      <string>&amp;Use relative paths</string>
                    </property>
                    <property name="toolTip" stdset="0" >
                      <string>Use relative paths instead of absolute</string>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>&lt;p>If this option is checked, the entries in the playlist will be relative to its location.
&lt;p>Example: If your playlist is located in &lt;em>/home/myself/music&lt;/em> and
your audio files are in &lt;em>/home/myself/music/cool&lt;/em>; then the entries in the
playlist will look something like: &lt;em>cool/track1.ogg&lt;/em>.</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item>
              <widget class="QCheckBox" name="m_checkSingleFile" >
                <property name="text" >
                  <string>Create si&amp;ngle file</string>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>Rip all tracks to a single file</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;p>If this option is checked K3b will create only one
audio file no matter how many tracks are ripped. This
file will contain all tracks one after the other.
&lt;p>This might be useful to rip a live album or a radio play.
&lt;p>&lt;b>Caution:&lt;/b> The file will have the name of the first track.</string>
                </property>
              </widget>
            </item>
            <item>
              <layout class="QHBoxLayout" >
                <item>
                  <spacer name="spacer1_2_2" >
                    <property name="sizeHint" >
                      <size>
                        <width>20</width>
                        <height>16</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item>
                  <widget class="QCheckBox" name="m_checkWriteCueFile" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="text" >
                      <string>Write &amp;cue file</string>
                    </property>
                    <property name="toolTip" stdset="0" >
                      <string>Write a cuefile</string>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>&lt;p>If this option is checked K3b will create a CDRWIN cue file which allows to easily write a copy of the audio CD on other systems.</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item row="1" column="0" >
        <widget class="Q3GroupBox" name="groupBox1" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
            </sizepolicy>
          </property>
          <property name="title" >
            <string>Target Folder</string>
          </property>
          <layout class="QGridLayout" >
            <item row="1" column="0" >
              <widget class="QLabel" name="textLabel1" >
                <property name="text" >
                  <string>Free space in directory:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="0" column="0" colspan="2" >
              <widget class="KUrlRequester" name="m_editBaseDir" />
            </item>
            <item row="1" column="1" >
              <widget class="QLabel" name="m_labelFreeSpace" >
                <property name="text" >
                  <string>-</string>
                </property>
                <property name="alignment" >
                  <set>Qt::AlignVCenter|Qt::AlignRight</set>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="0" >
              <widget class="QLabel" name="textLabel1_2" >
                <property name="text" >
                  <string>Space needed:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="1" >
              <widget class="QLabel" name="m_labelNeededSpace" >
                <property name="text" >
                  <string>-</string>
                </property>
                <property name="alignment" >
                  <set>Qt::AlignVCenter|Qt::AlignRight</set>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
    <customwidgets>
    <customwidget>
      <class>KComboBox</class>
      <extends>QComboBox</extends>
      <header location="global" >kcombobox.h</header>
    </customwidget>
  </customwidgets>
  <includes>
    <include location="local" >kurlrequester.h</include>
    <include location="local" >kpushbutton.h</include>
  </includes>
  <connections>
    <connection>
      <sender>m_checkCreatePlaylist</sender>
      <signal>toggled(bool)</signal>
      <receiver>m_checkPlaylistRelative</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
    <connection>
      <sender>m_checkSingleFile</sender>
      <signal>toggled(bool)</signal>
      <receiver>m_checkWriteCueFile</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
  </connections>
</ui>
